Client/src/Components/Inputs/Checkbox/index.jsx

```diff
@@ -40,6 +40,7 @@ const Checkbox = ({
 	/* TODO move sizes to theme */
 	const sizes = { small: "14px", medium: "16px", large: "18px" };
 	const theme = useTheme();
+	const override = typeof label == "string" ? {} : { alignSelf: "flex-start" };
```

Comment:
This makes the use of `CheckBox` unclear, you would not know that `alignSelf` is set to `flex-start` if the label is not a string unless you had a deep understanding of the implementation of `CheckBox`. 

The point of having the reusable components is that you should not need to know how they work, merely that you can use them.

If you need to be able to set `alignSelf` then it would be better to expose a prop to set it explicitly.  Please include JSdoc as well so that InteliSense can provide information on how to use the component.

Reply:
Fixed here: https://github.com/bluewave-labs/checkmate/pull/1306/commits/54c2826205c3aaa5413eb0394af044457c8b5fc4